### DISCUSSION TITLE
Class library documentation link is broken

### DISCUSSION THREAD

**Discussion by kabua** [ookii-dialogs/ookii-dialogs-wpf#113] [General]
The link, which is found on this page: http://www.ookii.org/Software/Dialogs/
Under 
```
Documentation
[Class library documentation](http://www.ookii.org/docs/dialogs-1.0)
```

is broken.

**Comment by augustoproiete** [comment]
Thanks @kabua. I maintain the project on GitHub, but don't have access to the website. Pinging @SvenGroot for awareness

**Reply by SvenGroot** [reply]
Thanks for bringing this to my attention. I've restored the class library documentation, but please be aware that this is still documentation for my original version of Ookii.Dialogs, and doesn't reflect any changes made in this repository.

I've also added a link to this GitHub from that page on my site, since people will probably be more interested in this version than my old, no longer maintained one.

Thanks for helping keep this project alive!
